spec/models/company_jobs_fetcher_spec.rb

```diff
@@ -3,7 +3,7 @@
 RSpec.describe CompanyJobsFetcher do
   context "with an existing company" do
     before do
-      allow($stdout).to receive(:write) # suppresses terminal clutter
+      # allow($stdout).to receive(:write) # suppresses terminal clutter
```

Comment:
What changed here?

Reply:
It was a lazy test that relied on capturing what's printed to the console ($stdout). One problem is that diverting $stdout means it doesn't display anymore, so you have no info on why the test failed. I changed this to evaluate method result rather than its printed output.